.travis.yml

```diff
@@ -4,8 +4,8 @@ go:
   - "1.13"
 script:
   - make
-  - npm install -g snyk
-  - make snyk
+  - 'if [ "$TRAVIS_PULL_REQUEST" = "false" ]; then npm install -g snyk; fi'
```

Comment:
can we just better do `SNYK_TOKEN != ""` that way for PRs from the same repo snyk runs.